docs/examples/cas9_kymotracking/cas9_kymotracking.rst

```diff
@@ -61,8 +61,7 @@ data looks like::
 
 .. image:: kymo_force.png
 
-Note how we specified a `vmax` for the image. This argument reflects the photon count that corresponds to the maximum
-of the colormap. Any photon count higher than that will be clipped to the maximal color.
+Note how we specified a color adjustment for the kymograph. Any photon count outside the range provided to :class:`~lumicks.pylake.ColorAdjustment` will be clipped to the nearest color.
```

Comment:
Makes sense to do an overhaul of the docs. I imagined it would take more time/effort to do it all at once - hence this "chipping off one at a time" approach. If the preferred way is the "tutorial voice" that's alright as well :ok_hand: